test/modules/EdgeHubRestartTester/EdgeHubRestartTester.csproj

```diff
@@ -27,14 +27,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices" Version="1.28.1-NestedEdge" />
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.33.1-NestedEdge" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.3" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="3.1.3" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.3" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.3" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="3.1.3" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.3" />
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.31.1" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.36.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="5.0.0" />
```

Comment:
Do we need to update Config packages to 5.0.0? What are the implications of doing that?